lib/include/public/Version.hpp

```diff
@@ -6,8 +6,8 @@
 #define MAT_VERSION_HPP
 // WARNING: DO NOT MODIFY THIS FILE!
 // This file has been automatically generated, manual changes will be lost.
-#define BUILD_VERSION_STR "3.7.62.1"
-#define BUILD_VERSION 3,7,62,1
+#define BUILD_VERSION_STR "3.7.240.1"
```

Comment:
Is there plan to create a new release along with the submodule update ? If yes, can you please update the PR title and description accordingly.

Reply:
> Is there plan to create a new release along with the submodule update ? If yes, can you please update the PR title and description accordingly.

Done